src/vanilla/atom.ts

```diff
@@ -40,6 +40,7 @@ type OnMount<Args extends unknown[], Result> = <
 export interface Atom<Value> {
   toString: () => string
   read: Read<Value>
+  is?: (a: Atom<unknown>) => boolean
```

Comment:
I thought about requiring this property rather than making it optional, but as we want to release this as a patch, let's do this.
Eventually, if it makes sense (like moving to `class` syntax), we can consider changing it a required property.

Reply:
A scenario needs to be considered: A global atom `anAtom` has two copies `copy1` and `copy2`, it is possible that `copy1` internally calls `set(copy2, value)`, so even with this change, we still needs to add another property in jotai-scope's intercepted copy to mark two copies as having the same original atom.

Reply:
So, does it mean it requires to be `is?: (a: Atom<unknown>, b: Atom<unknown>) => boolean` and doesn't use `this`?

Reply:
Well, for the jotai-scope PR, there is no need to do so.
Personally, I think the term should not be called `is`, as `is` suggests equality and equality is **symmetric**, while in this case it is not.
https://doc.rust-lang.org/std/cmp/trait.PartialEq.html